pkg/kds/context/context.go

```diff
@@ -83,10 +84,16 @@ func DefaultContext(
 			RemoveStatus()),
 		reconcile.If(
 			reconcile.And(
-				reconcile.ScopeIs(core_model.ScopeMesh),
```

Comment:
Why do we want to hash global-scoped resources now?

Reply:
To avoid conflicts. If a Global scoped resource can be applied on both global and zone CP, why wouldn't we hash it?

So far Global resources were
* GlobalSecret - we sync only signing key from global to zone
* Mesh - we sync as is without hashing
* Config - we sync selectively
* RBAC resources - we don't sync them